src/Marketplace/Az.Marketplace.psd1

```diff
@@ -0,0 +1,24 @@
+@{
+  GUID = '4db1f51a-b835-4783-be47-0b8939492ea5'
+  RootModule = './Az.Marketplace.psm1'
+  ModuleVersion = '1.1.0'
+  CompatiblePSEditions = 'Core', 'Desktop'
+  Author = 'Microsoft Corporation'
+  CompanyName = 'Microsoft Corporation'
+  Copyright = 'Microsoft Corporation. All rights reserved.'
+  Description = 'Microsoft Azure PowerShell: Marketplace cmdlets'
+  PowerShellVersion = '5.1'
+  DotNetFrameworkVersion = '4.7.2'
+  RequiredAssemblies = './bin/Az.Marketplace.private.dll'
+  FormatsToProcess = './Az.Marketplace.format.ps1xml'
+  FunctionsToExport = 'Copy-AzMarketplacePrivateStoreCollectionOffer', 'Get-AzMarketplaceBillingPrivateStoreAccount', 'Get-AzMarketplaceCollectionToSubscriptionMapping', 'Get-AzMarketplacePrivateStore', 'Get-AzMarketplacePrivateStoreCollection', 'Get-AzMarketplacePrivateStoreCollectionOffer', 'Get-AzMarketplaceQueryPrivateStoreOffer', 'New-AzMarketplacePrivateStore', 'New-AzMarketplacePrivateStoreCollection', 'New-AzMarketplacePrivateStoreCollectionOffer', 'Remove-AzMarketplacePrivateStoreCollection', 'Remove-AzMarketplacePrivateStoreCollectionOffer', 'Set-AzMarketplaceBulkPrivateStoreCollectionAction', 'Set-AzMarketplacePrivateStore', 'Set-AzMarketplacePrivateStoreCollection', 'Set-AzMarketplacePrivateStoreCollectionOffer', '*'
```

Comment:
Add `Get-AzMarketplaceCollectionPrivateStoreToSubscriptionMapping` here.

Reply:
its already there, the name of Cmdlet changed to "Get-AzMarketplaceCollectionToSubscriptionMapping"